src/apps/Profile/ProfileBillingPlanCancel.js

```diff
@@ -52,7 +52,9 @@ const ProfileBillingPlanCancel = Radium(React.createClass({
           There was an error sending your request.
         </div>
         <div>
-          Sorry about that. Please write us at <a href={`mailto:${SYNCANO_BILLING_EMAIL}`}>{SYNCANO_BILLING_EMAIL}</a>.
+          {'Sorry about that. Please write us at '}
```

Comment:
if you are going to make another commit - change this please, 'write us' is incorrect. I suggest contact us